source/src/molecule-unfolding/cdk/utils/custom-resource-lambda/create-event-rule/index.js

```diff
@@ -0,0 +1,114 @@
+const {
+    CloudFormationClient,
+    CreateStackCommand,
+    UpdateStackCommand,
+    DeleteStackCommand,
+    DescribeStacksCommand
+} = require("@aws-sdk/client-cloudformation");
+
+const {
+    readFileSync
+} = require('fs');
+
+exports.handler = async function (event, context) {
+    await _handler(event, context).then(() => {
+        console.log("=== complete ===")
+    }).catch((e) => {
+        console.log(e);
```

Comment:
change to `console.error`

Generally I recommend to use logger library like [this](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/logger/#getting-started)

Reply:
changed to use @aws-lambda-powertools/logger